lib/views/watch_live/watch_live.dart

```diff
@@ -111,15 +113,12 @@ class _ParticipantStreamScreenState extends State<ParticipantStreamScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        title: const Text(
-            'This is the stream title (same with home page)'), // const for now
-      ),
```

Comment:
I think it'd be nice to remove the AppBar, to give it more of a "full screen" feel 😏